classes/Context.php

```diff
@@ -453,4 +457,19 @@ protected function getTranslationResourcesDirectories()
 
         return $locations;
     }
+
+    /**
+     * Returns the computing precision according to the current currency
+     *
+     * @return int
+     */
+    public function getComputingPrecision()
+    {
+        if (empty($this->priceComputingPrecision)) {
```

Comment:
```suggestion
        if ($this->priceComputingPrecision === null) {
```
otherwise we can't have 0 as precision

Reply:
Damn, you're right, (and welcome back!)

Reply:
Still on vacation 🙄

Reply:
@PierreRambaud I fixed it, but I think I will revert to what I did initially,  because the value of priceComputingPrecision should never be zero, but at a minimum of two, so if for some weird reason something sets the value to zero, then we should consider it as if it was null and calculate its value again.

Reply:
Weird to disallow the zero precision :thinking:

Reply:
The idea is that the computing precision should always be higher than the display precision.
(The rule we chose for now is computing precision = display precision * 3)